src/main/java/net/minecraftforge/common/loot/LootModifierManager.java

```diff
@@ -0,0 +1,74 @@
+package net.minecraftforge.common.loot;
+
+import java.util.Collection;
+
+//import static net.minecraft.client.resources.JsonReloadListener.LOGGER;
+
+import java.util.Map;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableMap.Builder;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonObject;
+
+import net.minecraft.client.resources.JsonReloadListener;
+import net.minecraft.profiler.IProfiler;
+import net.minecraft.resources.IResourceManager;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.world.storage.loot.conditions.ILootCondition;
+import net.minecraft.world.storage.loot.conditions.LootConditionManager;
+import net.minecraft.world.storage.loot.functions.ILootFunction;
+import net.minecraft.world.storage.loot.functions.LootFunctionManager;
+import net.minecraftforge.fml.LogicalSide;
+import net.minecraftforge.fml.common.thread.EffectiveSide;
+import net.minecraftforge.registries.ForgeRegistries;
+
+public class LootModifierManager extends JsonReloadListener {
+	private static LootModifierManager instance;
```

Comment:
Tabs -> spaces

Reply:
All of these (tabs -> spaces) should be resolved now.